tests/linalg_test.py

```diff
@@ -1450,12 +1450,13 @@ def test_tridiagonal_solve(self, dtype):
                             for dtype in float_types + complex_types))
   @jtu.skip_on_devices("gpu", "tpu")
   def testSchur(self, shape, dtype):
-      rng = jtu.rand_default(self.rng())
-      n = shape[-1]
-      args_maker = lambda: [rng(shape, dtype)]
+      if jax._src.lib._xla_extension_version > 38:
```

Comment:
Rather than blocking the test code on an `iff` statement, it would be better to use `self.skipTest(reason)` so that the test is reported as skipped.